weblate/trans/machine/weblatetm.py

```diff
@@ -71,8 +109,8 @@ def download_translations(self, source, language, text, unit, user):
         """Download list of possible translations from a service."""
         matching_units = Unit.objects.more_like_this(unit)
 
-        return [
-            self.format_unit_match(munit, 50)
+        return list(set([
```

Comment:
Same as above, did you see need for `list(set())`?

Reply:
to remove potential duplicates